### PR TITLE
Added a callback function to handle import exception

### PR DESCRIPTION
**Pull request by gbenhaim**
ci test please

### CODE REVIEW

File: lago/plugins/__init__.py [hunk 1]
```diff
@@ -58,7 +61,12 @@ def load_plugins(namespace, instantiate=True):
     Returns:
         dict of str, object: Returns the list of loaded plugins
     """
-    mgr = ExtensionManager(namespace=namespace, )
+    mgr = ExtensionManager(namespace=namespace,
+                           on_load_failure_callback=(
+                               lambda manager, ep, err: LOGGER.debug(
```
Comment (david-caro) [inline lago/plugins/__init__.py]: Usually if you are not going to use one of the params, you set it to '_', like:

`lambda _, ep, err:...`

also, try to be consistent with the indenting (if you are breaking the line after the '(' then do so always).

Though as long as it passes the yapf check I'll merge ;)

**Comment by david-caro** [comment]
you can squash the last two commits in one (git rebase -i origin/master, and change the last from 'pick' to 'f', that will squash it into the previous one)

**Comment by gbenhaim** [comment]
ci :shipit:

**Comment by david-caro** [comment]
ci :shipit:

**Comment by david-caro** [comment]
ci :shipit:

**Pull request merged**